vaadin-platform-react-hybrid-test/src/test/java/com/vaadin/platform/react/test/FlowMainLayoutIT.java

```diff
@@ -16,13 +17,49 @@ public void hillaViewInFlowLayout() {
         waitUntil(ExpectedConditions.presenceOfElementLocated(
                 By.id("flow-main")));
 
-        // Navigate to Flow view
+        // Navigate to Hilla view
         getMenuElement("Hello React in Flow Layout").get().click();
 
         waitUntil(ExpectedConditions.presenceOfElementLocated(
                 By.id("flow-hilla")));
 
-        // navigate away from Flow view
+        // navigate away from Hilla view
+        getMenuElement("Flow Hello").get().click();
+
+        waitForElement("Should have navigated to HelloWorld Flow",
+                By.id("flow-hello"));
+    }
+
+    @Test
+    public void hillaViewWithFlowOptOut() {
```

Comment:
Both new tests are failing in CI, anything missing?

Reply:
the failure is expected as it needs the fix (https://github.com/vaadin/hilla/pull/2934).